### ISSUE TITLE
Сборка для Windows x32bit

### ISSUE THREAD

**Issue by Sithell** [DEgITx/rats-search#18]
Есть ли версия для 32-битных систем? На Пикабу есть только 64, не нашел инструкций по сборке в README. Заранее спасибо.

**Comment by DEgITx** [comment]
Пока только для 64-битных, может быть в ближайшее время будет и для 32-битных

**Comment by Sithell** [comment]
Я не могу собрать из исходников у себя на компьютере под 32? Или там есть какие-то ограничения?

**Comment by DEgITx** [comment]
да, там есть компонент под 64битную платформу бинарный, поэтому не собирете пока что.
В следующих версия наверное под обе платформы будет.

**Comment by Sithell** [comment]
Спасибо. Если не затруднит, отпишитесь как только/если появится под x32.
